packages/material-ui/src/Grow/Grow.d.ts

```diff
@@ -4,8 +4,27 @@ import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../transitions/transition';
 
 export interface GrowProps extends Omit<TransitionProps, 'timeout'> {
+  /**
+   * A single child content element.
+   */
+  children?: React.ReactElement<any, any>;
+  /**
+   * If `true`, show the component; triggers the enter or exit animation.
+   */
+  in?: boolean;
+  /**
+   */
+  onEnter?: TransitionProps['onEnter'];
+  /**
+   */
```

Comment:
Should we be explicit about the ignore statements?

```suggestion
  /**
   * @ignore
   */
```

Reply:
Not here because it will be picked up by IntelliSense. We want to avoid as much custom directives as possible.